src/graphics/context.rs

```diff
@@ -881,12 +881,7 @@ impl GraphicsContext {
                 }
             }
             FullscreenType::Desktop => {
-                window.set_fullscreen(None);
-                window.set_decorations(false);
-                if let Some(monitor) = window.current_monitor() {
-                    window.set_inner_size(monitor.size());
-                    window.set_outer_position(monitor.position());
-                }
+                window.set_fullscreen(Some(winit::window::Fullscreen::Borderless(None)));
```

Comment:
I could be mis-remembering, but I thought the above code was a workaround for some other problem. Maybe fullscreen issues in Windows?

Reply:
I don't think so, it has been like that since `ggez` use `winit`: https://github.com/ggez/ggez/pull/388/files#diff-35c3205180019dc523a7ec61de86d88ad0d0a9ab700191bb8e76590f3bf290f4R472-R487

Reply:
Fair enough! I may be remembering an old `winit` bug, unrelated to ggez.